cli/endpoints/batch/deploy-models/openai-embeddings/model/MLmodel

```diff
@@ -0,0 +1,19 @@
+flavors:
+  openai:
+    code: null
+    data: model.yaml
+    openai_version: 0.27.8
+  python_function:
+    data: model.yaml
+    env:
+      conda: conda.yaml
+      virtualenv: python_env.yaml
+    loader_module: mlflow.openai
+    python_version: 3.8.5
+mlflow_version: 2.5.1.dev0
+model_uuid: b9a39a71f54e41efbd83b8307294b4d8
+signature:
+  inputs: '[{"type": "string"}]'
```

Comment:
Does this still use MLflow's enforce_signature but without requiring any specific column name? Does it check the pandas.datatypes only then?

Reply:
That's what I meant in the comment above. Any single-column dataframe will make this signature to work :D